### PR TITLE
YLP-2358 Pending patient new design

### PR DESCRIPTION
**Pull request by TimCanu**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [ ] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

**Comment by TimCanu** [comment]
Sorry about the error in the HCP integration test. It should be removed as soon as any HCP of the team can manage patients invitations

File: packages/yourloops/test/integration/assert/patient-list.assert.ts [hunk 1]
```diff
@@ -530,3 +533,118 @@ export const checkMonitoringAlertsIconsInactiveForFirstPatient = async (): Promi
...
+  expect(pendingPatientList).toHaveTextContent('Eric ArdMay 17, 2023pending-patient@diabeloop.frResend inviteCancel')
+  expect(pendingPatientList).toHaveTextContent('Rows per page:101–1 of 1')
+
+  // We open the dialog to cancel a pending invitation
+  const cancelInviteButton = screen.getByRole('button', { name: 'Remove patient pending-patient@diabeloop.fr' })
+  await userEvent.click(cancelInviteButton)
+  const cancelInviteDialog = screen.getByRole('dialog')
+  expect(cancelInviteDialog).toHaveTextContent('Remove Pending Patient from A - MyThirdTeam - to be deleted')
+  expect(cancelInviteDialog).toHaveTextContent('Are you sure you want to remove Pending Patient from A - MyThirdTeam - to be deleted?')
+  expect(cancelInviteDialog).toHaveTextContent('You and the care team will no longer have access to their data.')
```
Comment (celia-rouis) [inline packages/yourloops/test/integration/assert/patient-list.assert.ts:568]: **💭 thought:** ‏ The text should be modified in the case of a pending invitation given that the access to the patient's data has not been granted yet

File: packages/yourloops/components/patient-list/pending-patient-list/pending-patient-list.hook.tsx [hunk 2]
```diff
@@ -65,54 +78,114 @@ export const usePendingPatientListHook = (props: PendingPatientListHookProps): P
...
+          return [
+            <Tooltip
+              key={params.row.id}
+              title={tooltipText}
+              aria-label={tooltipText}
+            >
+              <div data-testid="reinvite-patient-button">
+              <Button
+                data-action="reinvite-patient"
+                startIcon={<MailIcon />}
```
[the remark this answers is not in the document]
Comment (TimCanu) [reply packages/yourloops/components/patient-list/pending-patient-list/pending-patient-list.hook.tsx:133]: Why is it best?

[the remark this answers is not in the document]
Comment (celia-rouis) [reply packages/yourloops/components/patient-list/pending-patient-list/pending-patient-list.hook.tsx:133]: Actually I was wrong, the message icon should be used for the messages and the mail icon for the e-mails 😄

File: packages/yourloops/test/integration/assert/patient-list.assert.ts (hunk 1 above)
Comment (TimCanu) [reply packages/yourloops/test/integration/assert/patient-list.assert.ts:568]: I created a ticket for this: https://diabeloop.atlassian.net/browse/YLP-2398 (not complete yet)

File: locales/en/yourloops.json [hunk 3]
```diff
@@ -107,13 +107,15 @@
...
   "certified-professional-account": "Certified professional",
+  "change-columns-settings": "Change columns settings",
   "change-password-info": "By clicking this button, you will receive an e-mail allowing you to change your password.",
   "chat-footer-private": "Private",
   "chat-footer-reply": "Reply",
   "chat-footer-start-writing": "Start typing",
   "coefficient-of-variation": "CV ({{cbgLabel}})",
   "coefficient-of-variation-tooltip": "**CV (Coefficient of Variation):** The ratio of the standard deviation to the mean glucose. For any period greater than 1 day, we calculate the mean of daily CV.",
   "colon": "{{label}}:",
+  "columns-settings-cannot-changed-pending-tab": "Columns settings cannot be changed while on the pending tab",
```
Comment (celia-rouis) [inline locales/en/yourloops.json:118]: **🤓 nitpick:** ‏We could keep the same structure as the sentence for the filters

```json
"columns-settings-cannot-changed-pending-tab": "Columns settings cannot be changed on Pending tab",
```

**Review by celia-rouis — APPROVED** [review]

**Pull request merged**